core/src/com/po/fuck/model/Room.java

```diff
@@ -33,6 +32,12 @@ private enum State {
     @SuppressWarnings("unchecked")
     protected Managed<InvisibleWall> walls[] = new Managed[4];
 
+    Room (Vector2 tillingPosition) {
+        this.tillingPosition = tillingPosition;
+        BasicSpriteInfo info = BasicSpriteManager.getBasicSpriteInfo(this.getClass());
+        this.geometryData = new GeometryData(new PositionData(new Vector2(), 0), info.getFrameSize());
```

Comment:
> this.geometryData = new GeometryData(new PositionData(new Vector2(), 0), info.getFrameSize());

Maybe `this.geometryData = new GeometryData(new PositionData(), info.getFrameSize());` ?